presets/4.5/rates/ZoeFPV_3D_rates.txt

```diff
@@ -0,0 +1,94 @@
+#$ TITLE: ZoeFPV 3D Rate Bomb
+#$ FIRMWARE_VERSION: 4.3
+#$ FIRMWARE_VERSION: 4.4
+#$ FIRMWARE_VERSION: 4.5
+#$ CATEGORY: RATES
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: rates, freestyle, juicy, sbang, zoeFPV, zoe, 3d, upside down, sugarK
+#$ AUTHOR: ZoeFPV (co authored sugarK)
```

Comment:
prob better to just mention SugarK in the description...
In the case of future presets from ZoeFPV so we had just one author for dropdown.

Reply:
done